### PR TITLE
DPM-106 add new type ACTIVITY for OPEN_MODAL command

### PR DESCRIPTION
**Pull request by AndreasSaltsberg**
- Adds activities modal to OPEN_MODAL command
- Documentation in README and example

### CODE REVIEW

**Comment by pipedrive-bot-eventsink** [comment]
### Main branch dependencies
[snyk vulnerabilities](https://app.snyk.io/org/gravity-byw/project/5013564b-fbd2-498f-ad71-ca01748463df)

**Review by bashmach — APPROVED** [review]
LGTM

File: src/types.ts [hunk 1]
```diff
@@ -77,6 +78,20 @@ export type OrganizationModalAttributes = {
...
 
+export type ActivityModalAttributes = {
+	type: Modal.ACTIVITY;
+	prefill?: {
+		subject?: string;
+		dueDate?: string;
+		dueTime?: string;
+		duration?: string;
+		note?: string;
+		description?: string;
```
Comment (RuTsEST) [inline src/types.ts:89]: Maybe having this as `publicDescription` would be better to have consistency with what the modal expects from us?
```suggestion
		publicDescription?: string;
```

**Comment by pipedrive-bot-dora** [comment]
## Neodora Analysis ⚠


<details><summary><strong> ⚠ Unknown tribe name</strong></summary>
<br/>

**Message:** The property `owner` in `repository.json` must be a valid tribe name present in **mission-tracking**
**Repository file:** [repository.json](https://github.com/pipedrive/app-extensions-sdk/tree/c16c02b7008b69410eb28a25c3f04cdb49231dc8/repository.json)

<table>
<thead>
	<tr>
			<th>Owner status</th>
	</tr>
</thead>
<tbody>
		<tr>
				<td>'GRAVITY' is an unknown tribe name</td>
		</tr>
</tbody>
</table>


🔍 [How to fix it](https://neodora.pipedrive.tools/rules/tribeName)
<br/>
</details>

<details><summary><strong> ⚠ Missing cache in Github Workflow</strong></summary>
<br/>

**Message:** Github workflow should be using caching when step uses `actions/setup-node`

<table>
<thead>
	<tr>
			<th>Workflow file</th>
			<th>Job name</th>
			<th>Step index</th>
			<th>Actual cache value</th>
			<th>Expected cache value</th>
	</tr>
</thead>
<tbody>
		<tr>
				<td><a href="https://github.com/pipedrive/app-extensions-sdk/tree/c16c02b7008b69410eb28a25c3f04cdb49231dc8/.github/workflows/on-commit.yml">on-commit.yml</a></td>
				<td><code>lint</code></td>
				<td>2</td>
				<td><code>Not found</code></td>
				<td><code>npm</code></td>
		</tr>
</tbody>
</table>


🔍 [How to fix it](https://neodora.pipedrive.tools/rules/workflowCache)
<br/>
</details>


Commit [`c16c02b`](https://github.com/pipedrive/app-extensions-sdk/commit/c16c02b7008b69410eb28a25c3f04cdb49231dc8/)  Job [`95075`](https://jenkins.pipedrive.tools/job/neodora/95075/)

**Pull request merged**